frontend/components/common/other/AnimatedArrow.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+    <UpArrow :style="style" />
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import UpArrow from '~/assets/icons/uparrow.svg';
+
+type ArrowDirections = 'up' | 'down' | 'left' | 'right';
+
+export default Vue.extend({
+    components: {
+        UpArrow,
+    },
+    props: {
+        size: {
+            type: String,
+            default: '16px',
+        },
+        direction: {
+            type: String as Vue.PropType<ArrowDirections>,
+            default: 'up',
```

Comment:
Can we instead make it required? (I don't see where the component can be used without providing this prop)